ang/civicase/DashboardTab.html

```diff
@@ -11,6 +11,11 @@
       <div class="civicase__dashboard__tab__col civicase__dashboard__tab__col--right">
         <!-- New Cases -->
         <civicase-panel-query query="newCasesPanel.query" handlers="newCasesPanel.handlers" custom-data="newCasesPanel.custom">
+          <panel-query-actions>
+            <a class="panel-heading-control" ng-href="{{customData.viewCasesLink().url}}">
+              {{customData.viewCasesLink().label}}
```

Comment:
Why do we need to pass the `viewCasesLink` function as custom data and then get it back from the query panel when we can use it directly and avoid that pass -> get-back -> execute?

```html
<panel-query-actions>
  <a class="panel-heading-control" ng-href="{{viewCasesLink().url}}">
    {{viewCasesLink().label}}
  </a>
</panel-query-actions>
```